query-node/typegen.yml

```diff
@@ -0,0 +1,22 @@
+metadata:
```

Comment:
if this info is duplicated in `manifest.yml`, a separate `typegen.yml` file is not needed